numba_rvsdg/tests/test_scfg.py

```diff
@@ -65,28 +93,59 @@ def test_dict_conversion(self):
         # Case # 1: Acyclic graph, no back-edges
         cases = [
             {
-                "0": {"jt": ["1", "2"]},
-                "1": {"jt": ["3"]},
-                "2": {"jt": ["4"]},
-                "3": {"jt": ["4"]},
-                "4": {"jt": []},
+                "blocks": {
+                    "0": {"type": "basic"},
+                    "1": {"type": "basic"},
+                    "2": {"type": "basic"},
+                    "3": {"type": "basic"},
+                    "4": {"type": "basic"},
+                },
+                "edges": {
+                    "0": ["1", "2"],
+                    "1": ["3"],
+                    "2": ["4"],
+                    "3": ["4"],
+                    "4": [],
+                },
+                "backedges": {},
             },
             # Case # 2: Cyclic graph, no back edges
             {
-                "0": {"jt": ["1", "2"]},
-                "1": {"jt": ["5"]},
-                "2": {"jt": ["1", "5"]},
-                "3": {"jt": ["0"]},
-                "4": {"jt": []},
-                "5": {"jt": ["3", "4"]},
+                "blocks": {
+                    "0": {"type": "basic"},
+                    "1": {"type": "basic"},
+                    "2": {"type": "basic"},
+                    "3": {"type": "basic"},
+                    "4": {"type": "basic"},
+                    "5": {"type": "basic"},
+                },
+                "edges": {
+                    "0": ["1", "2"],
+                    "1": ["5"],
+                    "2": ["1", "5"],
+                    "3": ["1"],
```

Comment:
Can you explain this case, it goes from `0` to `1` in the diff?

Reply:
Same as https://github.com/numba/numba-rvsdg/pull/50#discussion_r1254183832 except in this case since graphs are random it doesn't matter where they point.